### DISCUSSION TITLE
Strongswan/swanctl on Rocky 9.2 linux

### DISCUSSION THREAD

**Discussion by alex-airspan** [strongswan/strongswan#1985] [Q&A]
Hi 

I try install Strongswan/swanctl on Rocky 9.2 linux

The dnf install not found the packets. Maybe you know how possible install / compile it ?

Thanks,
Alex

**Comment by tobiasbrunner** [comment]
You need to enable the EPEL repository (`dnf install epel-release`) to install the binary package.

See the docs in case you decide to [build from sources](https://docs.strongswan.org/docs/5.9/install/install.html#_compile_strongswan_yourself) instead.
